kerlescan/profile_parser.py

```diff
@@ -149,6 +149,30 @@ def _parse_tags(tags):
             name, vra = _get_name_vra_from_string(package)
             if name != "gpg-pubkey":
                 parsed_profile["installed_packages." + name] = vra
+            elif parsed_profile["installed_packages.gpg-pubkey"]:
```

Comment:
error is

```
ERROR: test_gpg_pubkey_parsing (test_info_parser.PackageParserTests)

----------------------------------------------------------------------

Traceback (most recent call last):

  File "/home/travis/build/RedHatInsights/kerlescan/tests/test_info_parser.py", line 48, in test_gpg_pubkey_parsing

    parsed_profiles = profile_parser.parse_profile(tests, "fake-name", None)

  File "/home/travis/build/RedHatInsights/kerlescan/kerlescan/profile_parser.py", line 152, in parse_profile

    elif parsed_profile["installed_packages.gpg-pubkey"]:

KeyError: 'installed_packages.gpg-pubkey'
```

I have a feeling you want to say `elif "installed_packages.gpg-pubkey" in parsed_profile`. For example:

```
>>> mydict = {"foo": None, "bar": "baz"}
>>> "foo" in mydict
True
>>> mydict["foo"]
>>> mydict["foo"] == None
True
```

The "in" operator will check for the existance of a key, but checking its value will bork if the key itself is not defined.